internal/testing/alldeps

```diff
@@ -41,28 +41,21 @@ github.com/gorilla/mux
 github.com/grpc-ecosystem/grpc-gateway
 github.com/hashicorp/errwrap
 github.com/hashicorp/go-cleanhttp
-github.com/hashicorp/go-hclog
```

Comment:
Why would `alldeps` change in this way?

Reply:
Good spot. Will revert the vault bump that caused this